lib/watir/locators/element/selector_builder/xpath.rb

```diff
@@ -3,32 +3,67 @@ module Locators
     class Element
       class SelectorBuilder
         class XPath
-          def initialize(should_use_label_element)
-            @should_use_label_element = should_use_label_element
-          end
-
-          def build(selectors)
-            adjacent = selectors.delete :adjacent
-            xpath = adjacent ? process_adjacent(adjacent) : './/*'
+          # Regular expressions that can be reliably converted to xpath `contains`
+          # expressions in order to optimize the locator.
+          CONVERTABLE_REGEXP = /
+          \A
```

Comment:
This and below should have +1 indentation level.

Reply:
man, I was hoping rubocop would automatically fix this for me when I ran it. :) Copied over the precise way it was in the other